README.md

```diff
@@ -100,10 +100,139 @@ This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0
         python $CONDA_PREFIX/scripts/isis3VarInit.py --data-dir=[path to data directory]  --test-dir=[path to test data directory]
 
 
-    More information about the ISIS3DATA environment variable and the ISIS3 Data Area can be found [here]("#The-ISIS3-Data-Area"). Now everytime the isis3 environment is activated, $ISISROOT, $ISIS3DATA, and $ISIS3TESTDATA will be set to the values passed to isis3VarInit.py. This does not happen retroactively, re-activate the isis3 envionment with one of the following commands:
+    More information about the ISIS3DATA environment variable and the ISIS Data Area can be found [here]("#The-ISIS3-Data-Area"). Now everytime the isis environment is activated, $ISISROOT, $ISIS3DATA, and $ISIS3TESTDATA will be set to the values passed to isis3VarInit.py. This does not happen retroactively, re-activate the isis envionment with one of the following commands:
+
+        for Anaconda 3.4 and up - conda activate isis
+        prior to Anaconda 3.4 - source activate isis
+
+
+### Practical Usage with other conda packages
+
+If you don't use conda for anything else on your computer, you can
+skip this section.
+
+If you use conda to install other packages, you may run into
+difficulties with adding the isis conda pacakge to those environments
+or adding other conda pacakges to the isis environment you just
+created above.  This is because the isis conda package pins a number
+of requirements that may clash with other packages.
+
+At this time, we recommend creating the isis environment as detailed
+above, and then not adding any other conda packages to it.  This
+is similar to the best practice usage of not adding any conda
+packages to your 'base' conda environment.
+
+Instead, when you need to have a conda environment with other
+packages that also needs to be able to run ISIS programs, we have
+two different options.  In both cases, we'll assume that you create
+a new environment called 'working' (but it could be named anything)
+that you want to add some conda packages to, but from which you
+also want ISIS access.
+
+The first step is to create 'working' and add whatever conda packages you want.
+
+#### Easy mode, with stacking
+
+1.  `conda activate isis`
+
+2.  `conda activate --stack working`
+
+That's it.  Told you it was easy.
+
+This activates the isis environment, gets it all set up, and
+then it 'stacks' the new working environment on top of it.  To get
+out, you'll have to `conda deactivate` two times to get out of
+`working` and then out of `isis`.
+
```

Comment:
the danger i smell here is, what happens when I do a "conda update --all" while stacked? Will it update (and maybe break) the isis env?

Reply:
I'm not sure, but that's a bit beyond the scope of these instructions.

I suspect that it will only affect the current environment, i.e. the one on the top of the 'stack,' and will not affect the isis environment that it is stacked on top of, but I haven't tested.

Reply:
Just tested, `conda update --all` only updates the packages in the current environment, that would be working in this example.